brain/CMakeLists.txt

```diff
@@ -49,8 +49,13 @@ set(BRAIN_SOURCES
 set(BRAIN_PUBLIC_INCLUDE_DIRECTORIES ${Boost_INCLUDE_DIRS})
 set(BRAIN_LINK_LIBRARIES
   PUBLIC Brion vmmlib
-  PRIVATE Keyv ${Boost_FILESYSTEM_LIBRARIES}
+  PRIVATE Lunchbox ${Boost_FILESYSTEM_LIBRARIES}
 )
+
+if(TARGET Keyv)
+  list(APPEND BRAIN_LINK_LIBRARIES Keyv)
```

Comment:
I find it better to repeat the PRIVATE keyword here than rely on context

Reply:
I did at first, but I found that it was not done ZeroEQ. I'll do it for both